.github/workflows/core-contracts-storage-check.yml

```diff
@@ -4,16 +4,18 @@ on:
   push:
     branches:
       - development
+      - fix-core-contracts-storage-check
   pull_request:
 
 jobs:
   provide_contracts:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           submodules: recursive
+          fetch-depth: 2  # Ensure we can access the previous commit
```

Comment:
Do we still need to fetch 2 latest commits since we compare PR with the development branch?

Reply:
I don't think so